src/main/java/seedu/address/model/account/Account.java

```diff
@@ -18,27 +21,32 @@
 
     // Identity fields
     private final Name name;
-    private final Transaction transaction;
+    private final List<Transaction> transactions;
 
     // Data fields
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Account(Name name, Transaction transaction, Set<Tag> tags) {
-        requireAllNonNull(name, transaction, tags);
+    public Account(Name name, List<Transaction> transactions, Set<Tag> tags) {
+        requireAllNonNull(name, transactions, tags);
         this.name = name;
-        this.transaction = transaction;
+        this.transactions = transactions;
```

Comment:
To maintain immutability, perhaps instantiate a new list (like how `tags` was instantiated), then do a `this.transactions.addAll(transactions);`

Reply:
Okay, will update this based on the latest version of Account (think this was based off of an older version)